py/server/deephaven/stream/__init__.py

```diff
@@ -11,6 +11,24 @@
 from deephaven.table import Table
 
 _JBlinkTableTools = jpy.get_type("io.deephaven.engine.table.impl.BlinkTableTools")
+_JAddOnlyToBlinkTableAdapter = jpy.get_type("io.deephaven.engine.table.impl.AddOnlyToBlinkTableAdapter")
+
+def add_only_to_blink(table: Table) -> Table:
+    """ Creates a blink table from the given add-only table.
+
+    Args:
+        table (Table): the source table
+
+    Returns:
+        a blink table
+
+    Raises:
+        DHError
+    """
+    try:
+        return Table(j_table=_JAddOnlyToBlinkTableAdapter.toBlink(table.j_table))
```

Comment:
`AddOnlyToBlinkTableAdapter` has the following block of code:
```
        if (!Boolean.TRUE.equals(table.getAttribute(Table.ADD_ONLY_TABLE_ATTRIBUTE))
                && !Boolean.TRUE.equals(table.getAttribute(Table.APPEND_ONLY_TABLE_ATTRIBUTE))) {
            throw new IllegalArgumentException("Argument table is neither add-only nor append-only");
        }
```

We should change this to make a `withAttributes` call to enforce add-only if those attributes are missing, instead of just throwing an IAE. That would make this easier to use.